api/types/container/responses-generated.go

```diff
@@ -0,0 +1,48 @@
+// Code generated by swagger-gen. DO NOT EDIT
+
+package container
+
+type ContainerChangesOKResponse []ContainerChangeResponseItem
+
+// change item in response to ContainerChanges operation
+type ContainerChangeResponseItem struct {
+	// Kind of change
+	Kind uint8 `json:"Kind"`
+	// Path to file that has changed
+	Path string `json:"Path"`
+}
+
+// OK response to ContainerCreate operation
+type ContainerCreateResponse struct {
+	// The ID of the created container
+	ID string `json:"Id"`
+	// Warnings encountered when creating the container
+	Warnings []string `json:"Warnings"`
+}
+
+// OK response to ContainerTop operation
+type ContainerTopResponse struct {
+	// Each process running in the container, where each is process is an array of values corresponding to the titles
+	Processes [][]string `json:"Processes,omitempty"`
+	// The ps column titles
+	Titles []string `json:"Titles,omitempty"`
+}
+
+// OK response to ContainerUpdate operation
+type ContainerUpdateResponse struct {
+	Warnings []string `json:"Warnings,omitempty"`
+}
+
+// OK response to ContainerWait operation
+type ContainerWaitResponse struct {
```

Comment:
Wondering what determines if this is named `ContainerWaitOKResponse` vs `ContainerWaitResponse`.

Although I like "response"; "body" may be more factual (i.e. it's not the full response); do you know where the code is to generate the names?

Basically; should we try to make the names match the old names?

Reply:
Comment doesn't look to be correct godoc as well